packages/ui/primitives/use-toast.ts

```diff
@@ -1,7 +1,8 @@
 // Inspired by react-hot-toast library
 import * as React from 'react';
 
-import { ToastActionElement, type ToastProps } from './toast';
+import type { ToastActionElement } from './toast';
+import { type ToastProps } from './toast';
```

Comment:
The auto lint seemed to mess this up, can you clean this by doing this?

```
import type { ToastActionElement, ToastProps } from './toast';
```

Reply:
sure, I'll fix it.